.github/workflows/test.yml

```diff
@@ -0,0 +1,60 @@
+name: Test
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+  push:
+    branches:
+      - develop
+      - trunk
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  wpa:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        snapshot: ["3593222a181f0f91824947dbd06985d5", "9ea863cebfcbb416c6334a6bef6552f0", "803989004f4b101a644ebfe9bb67e712"]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set PHP version
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.2'
+        coverage: none
+        tools: composer:v1
+
+    - name: Install dependencies
+      run: composer install
+
+    - name: Setup WP Snapshots
+      run: ./vendor/bin/wpsnapshots configure 10up --aws_key=${{ secrets.AWS_ACCESS_KEY_ID }} --aws_secret=${{ secrets.AWS_SECRET_ACCESS_KEY }} --user_name=wp-acceptance --user_email=travis@10up.com
```

Comment:
Is this email here still the correct one to use?

Reply:
Thanks for the head up! Fixed it.